modules/core/src/lib/layer.js

```diff
@@ -558,6 +565,7 @@ export default class Layer extends Component {
     // initializeState callback tends to clear state
     this.setChangeFlags({dataChanged: true, propsChanged: true, viewportChanged: true});
 
+    this._updateAnimationProps();
```

Comment:
kind of confusing here, we claim to check if needs updating here, but actually does the update.